### PR TITLE
DOC: trunc, floor, ceil, rint, fix should all link to each other

### PR DESCRIPTION
**Pull request by timhoffm**

### CODE REVIEW

File: numpy/core/code_generators/ufunc_docstrings.py [hunk 1]
```diff
@@ -1374,7 +1374,7 @@ def add_newdoc(place, name, doc):
 
     See Also
     --------
-    ceil, trunc, rint
+    ceil, trunc, rint, fix
 
     Notes
     -----
```
Comment (mattip) [inline numpy/core/code_generators/ufunc_docstrings.py:1380]: Could you also add a mention of `fix` to the note below?

Comment (timhoffm) [reply numpy/core/code_generators/ufunc_docstrings.py:1380]: Done.

File: numpy/core/code_generators/ufunc_docstrings.py [hunk 2]
```diff
@@ -1374,13 +1374,14 @@ def add_newdoc(place, name, doc):
...
-    Some spreadsheet programs calculate the "floor-towards-zero", in other
-    words ``floor(-2.5) == -2``.  NumPy instead uses the definition of
-    `floor` where `floor(-2.5) == -3`.
+    Other than the NumPy ``floor`` function, which always rounds down, the
+    FLOOR function in common spreadsheet programs calculates the
+    "floor-towards-zero", i.e. ``FLOOR(-2.5) == -2``. This "floor-towards-zero"
+    function is called ``fix`` in NumPy.
```
Comment (mattip) [inline numpy/core/code_generators/ufunc_docstrings.py:1384]:
```suggestion
    Some spreadsheet programs calculate the "floor-towards-zero", where
    ``floor(-2.5) == -2``.  NumPy instead uses the definition of
    `floor` where `floor(-2.5) == -3`. The "floor-towards-zero"
    function is called ``fix`` in NumPy.
```

**Review by mattip — APPROVED** [review]
LGTM.

**Comment by mattip** [comment]
Thanks @timhoffm

**Pull request merged**